<?xml version="1.0"?>

<launch>
  <include file="$(find camp_map)/launch/export.launch" >
    <arg name="map_in" value="map"/>
    <arg name="map_out" value="deca_map"/>
    <arg name="output_frame" value="deca"/>
  </include> 
</launch>